.github/workflows/pull-request.yml

```diff
@@ -58,6 +58,31 @@ jobs:
       - name: Run Typescript Type-Checker
         run: npm run typecheck
 
+  File-count-check:
+    name: Counting files changed
+    runs-on: ubuntu-latest
+    env:
+      BASE_BRANCH: ${{ github.event.pull_request.base.ref }}
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        
+      - name: Checkout code
+        run: git fetch origin pull/${{ github.event.pull_request.number }}/head:pr-branch
+        working-directory: ${{ github.workspace }}
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.11.6'
+
+      - name: Get changed files in base branch, compare difference, and execute python script
+        id: changed-files-base
+        run: |
+          git fetch origin $BASE_BRANCH
+          git checkout $BASE_BRANCH
+          python .github/workflows/check_files_submitted.py $BASE_BRANCH pr-branch
```

Comment:
Add the `base_branch` and `pr_branch` strings here too to make it very clear what is happening

Reply:
If I do it using '--', then they will no longer be positional arguments and therefore not mandatory. Therefore the script would become more complicated trying to handle the error cases manually, exceeding more than 130 lines. So I decided to provide more context using comments as to what exactly is happening. 

Let me know if you want it to be handled differently. Thank you.

Reply:
See the previous comment